scripts/prepare-release.sh

```diff
@@ -2,30 +2,50 @@
 set -e
 set -o pipefail
 
-PREVIOUS_VERSION=$(grep integreatly-operator deploy/olm-catalog/integreatly-operator/integreatly-operator.package.yaml | awk -F v '{print $2}')
+if [[ -z "$TYPE" ]]; then
+  TYPE="integreatly-operator"
```

Comment:
Should we be really specific here? i.e. if there's no entry default to integreatly-operator but if there is entry check that it is either integreatly-operator _or_ managed-api-service?

Reply:
I am not sure I fully understand the question been asked but here is what I think is the answer. 

The integreatly-operator is set as the default for now for two reasons.
1. The integreatly-operator still takes priority at least till the 2.7.1 has been released
2. There has been no release of the managed-api-service to date. I feel it would be hard set the default for something not existing yet

The question about checking for  integreatly-operator or managed-api-service, I feel where it would be a good idea there would be more work required which might be pushing the scope of the ticket a bit. It might be better to have sane defaults than trying to check every possibility.